2018/07/03 10:48:22 [error] [exception.ParseError] ParseError: syntax error, unexpected '$user' (T_VARIABLE) in C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\controllers\UserController.php:7
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(276): CWebApplication->createController('')
#1 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#2 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\base\CApplication.php(185): CWebApplication->processRequest()
#3 C:\xampp\htdocs\Zagadnienia\2_UserModel\index.php(13): CApplication->run()
#4 {main}
REQUEST_URI=/zagadnienia/2_UserModel/index.php?r=user
---
2018/07/03 10:48:48 [error] [exception.CHttpException.400] CHttpException: Your request is invalid. in C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php:337
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(282): CController->run('')
#4 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#5 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\base\CApplication.php(185): CWebApplication->processRequest()
#6 C:\xampp\htdocs\Zagadnienia\2_UserModel\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/zagadnienia/2_UserModel/index.php?r=user
---
2018/07/03 10:48:59 [error] [exception.CHttpException.400] CHttpException: Your request is invalid. in C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php:337
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(282): CController->run('')
#4 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#5 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\base\CApplication.php(185): CWebApplication->processRequest()
#6 C:\xampp\htdocs\Zagadnienia\2_UserModel\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/zagadnienia/2_UserModel/index.php?r=user
---
2018/07/03 10:49:00 [error] [exception.CHttpException.400] CHttpException: Your request is invalid. in C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php:337
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(282): CController->run('')
#4 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#5 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\base\CApplication.php(185): CWebApplication->processRequest()
#6 C:\xampp\htdocs\Zagadnienia\2_UserModel\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/zagadnienia/2_UserModel/index.php?r=user
---
2018/07/03 10:49:01 [error] [exception.CHttpException.400] CHttpException: Your request is invalid. in C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php:337
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(282): CController->run('')
#4 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#5 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\base\CApplication.php(185): CWebApplication->processRequest()
#6 C:\xampp\htdocs\Zagadnienia\2_UserModel\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/zagadnienia/2_UserModel/index.php?r=user
---
2018/07/03 10:49:07 [error] [exception.CHttpException.400] CHttpException: Your request is invalid. in C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php:337
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(282): CController->run('')
#4 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#5 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\base\CApplication.php(185): CWebApplication->processRequest()
#6 C:\xampp\htdocs\Zagadnienia\2_UserModel\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/zagadnienia/2_UserModel/index.php?r=user
---
2018/07/03 10:49:11 [error] [exception.CHttpException.400] CHttpException: Your request is invalid. in C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php:337
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(282): CController->run('')
#4 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#5 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\base\CApplication.php(185): CWebApplication->processRequest()
#6 C:\xampp\htdocs\Zagadnienia\2_UserModel\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/zagadnienia/2_UserModel/index.php?r=user
---
2018/07/03 10:49:12 [error] [exception.CHttpException.400] CHttpException: Your request is invalid. in C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php:337
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(282): CController->run('')
#4 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#5 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\base\CApplication.php(185): CWebApplication->processRequest()
#6 C:\xampp\htdocs\Zagadnienia\2_UserModel\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/zagadnienia/2_UserModel/index.php?r=user
---
2018/07/03 10:49:13 [error] [exception.CHttpException.400] CHttpException: Your request is invalid. in C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php:337
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(282): CController->run('')
#4 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#5 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\base\CApplication.php(185): CWebApplication->processRequest()
#6 C:\xampp\htdocs\Zagadnienia\2_UserModel\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/zagadnienia/2_UserModel/index.php?r=user
---
2018/07/03 10:49:13 [error] [exception.CHttpException.400] CHttpException: Your request is invalid. in C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php:337
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(282): CController->run('')
#4 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#5 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\base\CApplication.php(185): CWebApplication->processRequest()
#6 C:\xampp\htdocs\Zagadnienia\2_UserModel\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/zagadnienia/2_UserModel/index.php?r=user
---
2018/07/03 10:50:08 [error] [exception.CHttpException.400] CHttpException: Your request is invalid. in C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php:337
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(282): CController->run('')
#4 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#5 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\base\CApplication.php(185): CWebApplication->processRequest()
#6 C:\xampp\htdocs\Zagadnienia\2_UserModel\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/zagadnienia/2_UserModel/index.php?r=user
---
2018/07/03 10:50:32 [error] [exception.CHttpException.400] CHttpException: Your request is invalid. in C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php:337
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(282): CController->run('')
#4 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#5 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\base\CApplication.php(185): CWebApplication->processRequest()
#6 C:\xampp\htdocs\Zagadnienia\2_UserModel\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/zagadnienia/2_UserModel/index.php?r=user
---
2018/07/03 10:57:03 [error] [exception.ArgumentCountError] ArgumentCountError: Too few arguments to function User::__construct(), 1 passed in C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\controllers\UserController.php on line 7 and exactly 2 expected in C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\models\User.php:14
Stack trace:
#0 C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\controllers\UserController.php(7): User->__construct('maciek')
#1 [internal function]: UserController->actionIndex('maciek')
#2 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(UserController), Array)
#3 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UserController), Object(ReflectionMethod), Array)
#4 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(282): CController->run('')
#8 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#9 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\Zagadnienia\2_UserModel\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/zagadnienia/2_UserModel/index.php?r=user&name=maciek
---
2018/07/03 10:57:32 [error] [php] Object of class User could not be converted to string (C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\views\user\index.php:3)
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(872): UserController->renderFile()
#1 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(785): UserController->renderPartial()
#2 C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\controllers\UserController.php(8): UserController->render()
#3 unknown(0): UserController->actionIndex()
#4 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs()
#5 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#6 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(286): UserController->runAction()
#8 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(265): UserController->runActionWithFilters()
#9 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(282): UserController->run()
#10 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\base\CApplication.php(185): CWebApplication->processRequest()
#12 C:\xampp\htdocs\Zagadnienia\2_UserModel\index.php(13): CWebApplication->run()
REQUEST_URI=/zagadnienia/2_UserModel/index.php?r=user&name=maciek
in C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\views\user\index.php (3)
in C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\controllers\UserController.php (8)
in C:\xampp\htdocs\Zagadnienia\2_UserModel\index.php (13)
2018/07/03 10:57:39 [error] [php] Object of class User could not be converted to string (C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\views\user\index.php:3)
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(872): UserController->renderFile()
#1 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(785): UserController->renderPartial()
#2 C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\controllers\UserController.php(8): UserController->render()
#3 unknown(0): UserController->actionIndex()
#4 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs()
#5 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#6 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(286): UserController->runAction()
#8 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(265): UserController->runActionWithFilters()
#9 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(282): UserController->run()
#10 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\base\CApplication.php(185): CWebApplication->processRequest()
#12 C:\xampp\htdocs\Zagadnienia\2_UserModel\index.php(13): CWebApplication->run()
REQUEST_URI=/zagadnienia/2_UserModel/index.php?r=user&name=maciek
in C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\views\user\index.php (3)
in C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\controllers\UserController.php (8)
in C:\xampp\htdocs\Zagadnienia\2_UserModel\index.php (13)
2018/07/03 11:06:30 [error] [exception.CHttpException.400] CHttpException: Your request is invalid. in C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php:337
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(282): CController->run('')
#4 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#5 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\base\CApplication.php(185): CWebApplication->processRequest()
#6 C:\xampp\htdocs\Zagadnienia\2_UserModel\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/zagadnienia/2_UserModel/index.php?r=user&name=maciek
---
2018/07/03 11:06:31 [error] [exception.CHttpException.400] CHttpException: Your request is invalid. in C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php:337
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(282): CController->run('')
#4 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#5 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\base\CApplication.php(185): CWebApplication->processRequest()
#6 C:\xampp\htdocs\Zagadnienia\2_UserModel\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/zagadnienia/2_UserModel/index.php?r=user&name=maciek
---
2018/07/03 11:07:44 [error] [exception.Error] Error: Call to undefined function getLastElementKey() in C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\models\User.php:16
Stack trace:
#0 C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\controllers\UserController.php(7): User->__construct('maciek')
#1 [internal function]: UserController->actionIndex('maciek')
#2 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(UserController), Array)
#3 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UserController), Object(ReflectionMethod), Array)
#4 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(282): CController->run('')
#8 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#9 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\Zagadnienia\2_UserModel\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/zagadnienia/2_UserModel/index.php?r=user&name=maciek
---
2018/07/03 11:07:56 [error] [php] end() expects parameter 1 to be array, null given (C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\models\User.php:47)
Stack trace:
#0 C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\controllers\UserController.php(7): User->__construct()
#1 unknown(0): UserController->actionIndex()
#2 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs()
#3 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#4 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(286): UserController->runAction()
#6 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(265): UserController->runActionWithFilters()
#7 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(282): UserController->run()
#8 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\Zagadnienia\2_UserModel\index.php(13): CWebApplication->run()
REQUEST_URI=/zagadnienia/2_UserModel/index.php?r=user&name=maciek
in C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\models\User.php (47)
in C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\models\User.php (16)
in C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\controllers\UserController.php (7)
2018/07/03 11:08:09 [error] [php] end() expects parameter 1 to be array, null given (C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\models\User.php:47)
Stack trace:
#0 C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\controllers\UserController.php(7): User->__construct()
#1 unknown(0): UserController->actionIndex()
#2 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs()
#3 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#4 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(286): UserController->runAction()
#6 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(265): UserController->runActionWithFilters()
#7 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(282): UserController->run()
#8 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\Zagadnienia\2_UserModel\index.php(13): CWebApplication->run()
REQUEST_URI=/zagadnienia/2_UserModel/index.php?r=user&name=maciek
in C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\models\User.php (47)
in C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\models\User.php (16)
in C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\controllers\UserController.php (7)
2018/07/03 11:08:11 [error] [php] end() expects parameter 1 to be array, null given (C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\models\User.php:47)
Stack trace:
#0 C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\controllers\UserController.php(7): User->__construct()
#1 unknown(0): UserController->actionIndex()
#2 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs()
#3 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#4 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(286): UserController->runAction()
#6 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(265): UserController->runActionWithFilters()
#7 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(282): UserController->run()
#8 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\Zagadnienia\2_UserModel\index.php(13): CWebApplication->run()
REQUEST_URI=/zagadnienia/2_UserModel/index.php?r=user&name=maciek
in C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\models\User.php (47)
in C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\models\User.php (16)
in C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\controllers\UserController.php (7)
2018/07/03 11:08:32 [error] [exception.CDbException] CDbException: The table "User" for active record class "User" cannot be found in the database. in C:\xampp\htdocs\yii-1.1.19.5790cb\framework\db\ar\CActiveRecord.php:2390
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\db\ar\CActiveRecord.php(413): CActiveRecordMetaData->__construct(Object(User))
#1 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\db\ar\CActiveRecord.php(138): CActiveRecord->getMetaData()
#2 C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\models\User.php(47): CActiveRecord->__get('users')
#3 C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\models\User.php(16): User->getLastElementKey()
#4 C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\controllers\UserController.php(7): User->__construct('maciek')
#5 [internal function]: UserController->actionIndex('maciek')
#6 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(UserController), Array)
#7 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UserController), Object(ReflectionMethod), Array)
#8 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(282): CController->run('')
#12 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#13 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\base\CApplication.php(185): CWebApplication->processRequest()
#14 C:\xampp\htdocs\Zagadnienia\2_UserModel\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/zagadnienia/2_UserModel/index.php?r=user&name=maciek
---
2018/07/03 11:08:47 [error] [php] Undefined variable: users (C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\models\User.php:48)
Stack trace:
#0 unknown(0): UserController->actionIndex()
#1 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs()
#2 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#3 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(308): CInlineAction->runWithParams()
#4 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(286): UserController->runAction()
#5 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(265): UserController->runActionWithFilters()
#6 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(282): UserController->run()
#7 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(141): CWebApplication->runController()
#8 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\Zagadnienia\2_UserModel\index.php(13): CWebApplication->run()
REQUEST_URI=/zagadnienia/2_UserModel/index.php?r=user&name=maciek
in C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\models\User.php (48)
in C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\models\User.php (16)
in C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\controllers\UserController.php (7)
2018/07/03 11:08:55 [error] [exception.CDbException] CDbException: The table "User" for active record class "User" cannot be found in the database. in C:\xampp\htdocs\yii-1.1.19.5790cb\framework\db\ar\CActiveRecord.php:2390
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\db\ar\CActiveRecord.php(413): CActiveRecordMetaData->__construct(Object(User))
#1 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\db\ar\CActiveRecord.php(138): CActiveRecord->getMetaData()
#2 C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\models\User.php(42): CActiveRecord->__get('users')
#3 C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\controllers\UserController.php(8): User->browseUsers()
#4 [internal function]: UserController->actionIndex('maciek')
#5 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(UserController), Array)
#6 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UserController), Object(ReflectionMethod), Array)
#7 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(282): CController->run('')
#11 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#12 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\base\CApplication.php(185): CWebApplication->processRequest()
#13 C:\xampp\htdocs\Zagadnienia\2_UserModel\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/zagadnienia/2_UserModel/index.php?r=user&name=maciek
---
2018/07/03 11:09:11 [error] [exception.Error] Error: Undefined class constant 'users' in C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\models\User.php:42
Stack trace:
#0 C:\xampp\htdocs\Zagadnienia\2_UserModel\protected\controllers\UserController.php(8): User->browseUsers()
#1 [internal function]: UserController->actionIndex('maciek')
#2 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(UserController), Array)
#3 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UserController), Object(ReflectionMethod), Array)
#4 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(282): CController->run('')
#8 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#9 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\Zagadnienia\2_UserModel\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/zagadnienia/2_UserModel/index.php?r=user&name=maciek
---
2018/07/03 12:22:41 [error] [exception.CHttpException.400] CHttpException: Your request is invalid. in C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php:337
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(282): CController->run('')
#4 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#5 C:\xampp\htdocs\yii-1.1.19.5790cb\framework\base\CApplication.php(185): CWebApplication->processRequest()
#6 C:\xampp\htdocs\Zagadnienia\2_UserModel\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/zagadnienia/2_UserModel/index.php?r=user
---
